sys/net/application_layer/dhcpv6/_dhcpv6.h

```diff
@@ -349,6 +350,39 @@ typedef struct __attribute__((packed)) {
 void dhcpv6_client_dns_rns_conf(const dhcpv6_opt_dns_rns_t *opt,
                                 uint16_t netif);
 
+/**
+ * @brief   Find upstream network interface
+ *
+ * Either the one network interface configured at compile-time with @ref
+ * CONFIG_GNRC_DHCPV6_CLIENT_6LBR_UPSTREAM is picked or the first network
+ * interface that is not a 6LoWPAN interfaces if
+ * `CONFIG_GNRC_DHCPV6_CLIENT_6LBR_UPSTREAM` is 0.
+ *
+ * @return  The upstream network interface.
+ */
+gnrc_netif_t *_dhcpv6_client_simple_pd_find_upstream_netif(void);
```

Comment:
Why make those helper public instead of also moving them to `client.c`?

Reply:
For me it would also seem to be okay to merge both C files entirely.